raw_description/urdf/vacuum_gripper/gripper.urdf.xacro

```diff
@@ -2,70 +2,91 @@
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro">
 
   <xacro:include filename="$(find cob_description)/urdf/common.xacro"/>
+  <xacro:include filename="$(find raw_description)/urdf/vacuum_gripper/gripper.transmission.xacro" />
   <xacro:include filename="$(find raw_description)/urdf/vacuum_gripper/gripper.gazebo.xacro" />
 
   <xacro:property name="gripper_length" value="0.14"/>
   <xacro:property name="gripper_width" value="0.2"/>
   <xacro:property name="gripper_height" value="0.14"/>
   <xacro:property name="gripper_body_height" value="0.14"/>
-  <xacro:property name="gripper_radius" value="0.005"/>
+  <xacro:property name="gripper_radius" value="0.01"/>
 
   <xacro:macro name="vacuum_gripper" params="name parent *origin">
 
-    <!-- joint between arm_ee_link and vacuum_gripper_link -->
-    <joint name="${name}_base_joint" type="fixed">
-        <xacro:insert_block name="origin"/>
-        <parent link="${parent}"/>
-        <child link="${name}_frame"/>
-    </joint>
+  <!-- joint between arm_ee_link and vacuum_gripper_link -->
+  <joint name="${name}_base_joint" type="fixed">
+    <xacro:insert_block name="origin"/>
+    <parent link="${parent}"/>
+    <child link="${name}_frame"/>
+  </joint>
```

Comment:
only thing I dislike is that you removed one level of indentation within the ```<xacro:macro name="vacuum_gripper" params="name parent *origin">....<xacro:macro/>``` tags...